README.md

```diff
@@ -1,6 +1,46 @@
-# Hello World Project
-Simple website build with HTML and CSS
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello World
+
+> Html and CSS website 
+
+## Built With
 
-### Built with
 - HTML
-- CSS 
+- CSS
+
+To get a local copy up and running follow these simple example steps.
+
+### Setup
+- Clone this repo
```

Comment:
- [ ] [OPTIONAL] As the previous code reviewer did not mention this, I kindly suggest that you add a more descriptive setup instruction

### EXAMPLE
```
### Setup
If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.

[Download the ZIP from this location](https://github.com/c3c1l1a/Hello-world/archive/refs/heads/master.zip), or run the following [git](https://git-scm.com/downloads) command to clone the files to your machine:

```bash
git clone https://github.com/c3c1l1a/Hello-world.git
```

```
- Once the files are on your machine, open the _Hello-world_ folder in [Visual Studio Code](https://code.visualstudio.com/).

- With the files open in Visual Studio Code, press the **Go Live** button at the bottom of the window to launch the files with [Live Server](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer).
```
```